gateleen-monitoring/src/main/java/org/swisspush/gateleen/monitoring/RedisMonitor.java

```diff
@@ -19,7 +19,7 @@
 public class RedisMonitor {
 
     private Vertx vertx;
```

Comment:
private Vertx vertx;
private RedisAPI redisAPI;
private RedisProvider redisProvider;
private int period;
private long timer;
private Logger log = LoggerFactory.getLogger(RedisMonitor.class);

these variables could be marked as final.